src/doc/The_Configuration_File.md

```diff
@@ -66,10 +71,11 @@ This section configures the HTTP server:
 - `certificate`: path to the SSL certificate (*mandatory*).
 - `private-key`: path to the SSL private-key (*mandatory*).
 - `port`: port to listen on (*mandatory*).
-- `authorized-tokens-path`: path to the file giving the authentication tokens
+- `authorized-tokens-path`: path to the file of the authentication tokens
 that the server accepts (*mandatory-ish* since
-authentication-less server-side is not implemented so far). The format of the
-file is the SSH `authorized_keys` style.
+authentication-less server-side is not implemented so far). This file is in the
+SSH [`authorized_keys`](http://en.wikibooks.org/wiki/OpenSSH/Client_Configuration_Files#.7E.2F.ssh.2Fauthorized_keys)
+format.
```

Comment:
Note for later: we should make sure it is the *same* format and not something somewhat similar.